src/index.html

```diff
@@ -155,17 +168,51 @@ <h2 class="modal__title">New post</h2>
             type="button"
             class="modal__close-btn modal__close-btn_type_preview"
           ></button>
-          <img
-            class="modal__image"
-            id="preview__image"
-            src="  https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/7-photo-by-griffin-wooldridge-from-pexels.jpg"
-            alt=""
-          />
+          <img class="modal__image" id="preview__image" alt="" />
           <p class="modal__caption" id="preview__caption">Golden Gate bridge</p>
         </div>
       </div>
+      <div class="modal" id="delete__card-modal">
+        <div class="modal__container" data-type="delete-confirmation">
+          <button
+            type="button"
+            class="modal__close-btn modal__close-btn_type_delete"
+          ></button>
+          <h2 class="modal__title">
+            Are you sure you want to delete this message?
+          </h2>
+          <form class="modal__form">
+            <button
+              type="submit"
+              class="modal__button modal__button_type_delete"
+            ></button>
+            <button
+              type="button"
+              class="modal__button modal__button_type_cancel"
+            >
+              Cancel
+            </button>
+          </form>
+        </div>
+      </div>
+    </div>
+    <div class="modal" id="avatar__edit-modal">
+      <div class="modal__container">
+        <button type="button" class="modal__close-btn"></button>
+        <h2 class="modal__title">Edit profile avatar</h2>
+        <form name="edit__avatar-form" class="modal__form">
+          <label for="profile__avatar-input" class="modal__label"
+            >Profile Picture
+            <input
```

Comment:
* The `avatar` submit button should be disabled initially if the input is empty https://snipboard.io/U3sQCI.jpg 


Please, add a `required` attribute to the `avatar` input, to disable the button if the input is empty